projects/batfish/src/test/java/org/batfish/main/BatfishTest.java

```diff
@@ -176,6 +180,43 @@ public void testParseTopologyJson() throws IOException {
     assertEquals(topology.getEdges().size(), 2);
   }
 
+  @Test
+  public void testCheckValidTopology() throws IOException {
+    Batfish batfish = initBatfish();
```

Comment:
I don't think you need this object any more. The only thing you use it for is to call `checkTopology`, and that's a static function. So you can just do `Batfish.checkTopology`.